modules/services/hound.nix

```
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.hound;
+
+  configFile = pkgs.writeText "hound-config.json" (
+    builtins.toJSON {
+      inherit (cfg) max-concurrent-indexers dbpath repos;
+      health-check-url = "/healthz";
+    });
+
+  houndOptions = [
+    "--addr ${cfg.addr}"
+    "--conf ${configFile}"
+  ];
+
+  execStart = pkgs.writeScript "houndd-execstart" ''
+    #!${pkgs.runtimeShell}
+    export PATH="${pkgs.mercurial}/bin:${pkgs.git}/bin''${PATH:+:}$PATH"
+    exec ${cfg.package}/bin/houndd ${concatStringsSep " " houndOptions}
+  '';
+
+in
+
+{
+  meta.maintainers = [ maintainers.adisbladis ];
+
+  options.services.hound = {
+    enable = mkEnableOption "hound";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.hound;
+      defaultText = "pkgs.hound";
+      example = literalExample "pkgs.hound";
+      description = "The package to use for the hound binary.";
+    };
+
+    max-concurrent-indexers = mkOption {
+      type = types.int;
+      default = 2;
+      description = "Limit the amount of concurrent houndd indexers";
+    };
+
+    dbpath = mkOption {
+      type = types.string;
+      default = "${config.xdg.dataHome}/hound";
+      description = "The houndd database path";
+    };
+
+    addr = mkOption {
+      type = types.string;
+      default = "localhost:6080";
+      description = "Houndd listen address";
+    };
+
+
+    repos = mkOption {
+      type = types.attrs;
+      default = {};
+      description = "The houndd repos configuration";
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ cfg.package ];
+
+    systemd.user.services.hound = {
+      Unit = {
+        Description = cfg.package.meta.description;
+      };
+
+      Install = {
+        WantedBy = [ "default.target" ];
+      };
+
+      Service = {
+        Type = "simple";
+        ExecStart = "${execStart}";
```

As per suggestion above
```suggestion
        Environment = "PATH=${makeBinPath [ pkgs.mercurial pkgs.git ]}";
        ExecStart = "${cfg.package}/bin/houndd ${concatStringsSep " " houndOptions}";
```